package.json

```diff
@@ -25,6 +25,7 @@
     "request": "^2.81.0"
   },
   "devDependencies": {
-    "nodemon": "^1.12.1"
+    "nodemon": "^1.12.1",
+    "standard": "*"
```

Comment:
What is the utility of the "standard" dependency ?

Reply:
It's a linter and code fixer, it works without configuration.

https://github.com/standard/standard